### PR TITLE
TestCafe setup

### PR DESCRIPTION
**Pull request by jorilindell**
## Description :sparkles:
TestCafe setup
Test change language feature

### CODE REVIEW

File: testcafe/e2e/changeLanguage.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+import { header } from '../selectors/header';
+import { getPathname } from '../utils/clientUtils';
+import { getEnvUrl } from '../utils/settings';
+
+fixture('Landing page').page(getEnvUrl('/fi/home'));
+
+test('Changing language on landing page', async (t) => {
+  await t
+    .expect(screen.getAllByRole('link', { name: /etsi tekemistä/i }).exists)
```
Comment (Sartonon) [inline testcafe/e2e/changeLanguage.ts:11]: are there many links with text "etsi tekemistä"?

Comment (Sartonon) [reply testcafe/e2e/changeLanguage.ts:11]: if there is one, maybe getByRole is better. If there are many, could the the amount of test be tested?

**Review by Sartonon — APPROVED** [review]
Looks good. One question on getAllBy* usage :)

File: testcafe/e2e/changeLanguage.ts (hunk 1 above)
Comment (jorilindell) [reply testcafe/e2e/changeLanguage.ts]: Both footer and header have those links. I started testing exact amount of elements e.g. 
```
await t
    .expect(screen.getAllByRole('link', { name: /etsi tekemistä/i }).count)
    .eql(2)
    .expect(screen.getAllByRole('link', { name: /suosittelemme/i }).count)
    .eql(2);
```

**Pull request merged**